content/pt/docs/concepts/scheduling/kube-scheduler.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Escalonador do Kubernetes
+date: 2020-04-19
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+No Kubernetes, _escalonamento_ refere-se a garantir que os {{< glossary_tooltip text="Pods" term_id="pod" >}}
+sejam correspondidos aos {{< glossary_tooltip text="Nodes" term_id="node" >}} para que o
+{{< glossary_tooltip text="Kubelet" term_id="kubelet" >}} possa executá-los.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Visão geral do Escalonamento {#scheduling}
+
+Um escalonador observa Pods recém-criados que não possuem um Node atribuído. 
+Para cada Pod que o planejador descobre, ele se torna responsável por 
+encontrar o melhor Node para execução do Pod. O escalonador chega a essa decisão de alocação levando em consideração os princípios de programação descritos abaixo.
+
+Se você quiser entender por que os Pods são alocados em um Node específico 
+ou se planeja implementar um escalonador personalizado, esta página ajudará você a 
+aprender sobre escalonamento.
+
+## kube-scheduler
+
+[kube-scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/)
+é o escalonador padrão do Kubernetes e é executado como parte do 
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}}.
+O kube-scheduler foi projetado para que, se você quiser e precisar, possa
+escrever seu próprio componente de escalonamento e usá-lo.
+
+Para cada Pod recém-criado ou outros Pods não escalonados, o kube-scheduler 
+seleciona um Node ideal para execução. No entanto, todos os contêineres nos Pods 
+têm requisitos diferentes de recursos e cada Pod também possui requisitos diferentes. 
+Portanto, os Nodes existentes precisam ser filtrados de acordo com os requisitos de 
+escalonamento específicos.
+
+Em um cluster, Nodes que atendem aos requisitos de escalonamento para um Pod
+são chamados de Nodes _viáveis_. Se nenhum dos Nodes for adequado, o Pod
+permanece não escalonado até que o escalonador possa alocá-lo.
+
+O escalonador encontra Nodes viáveis para um Pod e, em seguida, executa um conjunto de
+funções para pontuar os Nodes viáveis e escolhe um Node com a maior
+pontuação entre os possíveis para executar o Pod. O escalonador então notifica
+o servidor da API sobre essa decisão em um processo chamado _binding_.
+
+Fatores que precisam ser levados em consideração para decisões de escalonamento incluem
+requisitos individuais e coletivos de recursos,
+restrições de política hardware / software, especificações de afinidade e anti-afinidade,
+localidade de dados, interferência entre cargas de trabalho e assim por diante.
+
+
+### Seleção do Node no kube-scheduler {#kube-scheduler-implementation}
```

Comment:
```suggestion
### Seleção do Node no kube-scheduler {#kube-scheduler-implementação}
```
?

Reply:
I guess we should keep the domain name in this case as `kube-scheduler-implementation`

Reply:
There's really no need. `kube-scheduler-implementation` is not a domain name.

Reply:
how about use  `implementação-kube-scheduler` ?

Reply:
Fine by me.

Reply:
Done